### DISCUSSION TITLE
No traceId after Flowable awake

### DISCUSSION THREAD

**Discussion by zhangzx1996** [apache/skywalking#12565] [Q&A]
I used Flowable as a workflow microservice. When service A initiates a request that requires approval, Flowable will pause the request and wake up to continue execution when it is approved.

I found the traceId is disappear after awake.  what can I fix this? very 3Q!

**Comment by wu-sheng** [comment]
That depends. We never faced a service that would pause an RPC call. Based on what you shared, you need a plugin at Flowable side, which needs to hold a snapshot of tracing context in memory or in Flowable storage, and restore it when the request is permitted.

But, to be honest, we don't have experience about this.

**Reply by wu-sheng** [reply]
> When service A initiates a request that requires approval, Flowable will pause the request and wake up to continue execution when it is approved.

If this is done manually, how do you make sure it is efficient? I am not following the use case actually.

**Reply by zhangzx1996** [reply]
I don't need the traceId to be the one that initiated the request, but the current situation is N/A.

**Reply by wu-sheng** [reply]
N/A just means no trace context. Nothing special. This is very common.
